Ktisis/Structs/Actor/EquipmentSet.cs

```diff
@@ -0,0 +1,208 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using Dalamud.Logging;
+using FFXIVClientStructs.FFXIV.Client.Game;
+using FFXIVClientStructs.FFXIV.Client.UI.Misc;
+using Ktisis.GameData;
+using Ktisis.GameData.Excel;
+
+namespace Ktisis.Structs.Actor {
+	public struct EquipmentSet
+	{
+		public int ID;
+		public SetSource Source;
+		public string Name;
+		public EquipmentSet(int iD, string name, SetSource source)
+		{
+			ID = iD;
+			Name = name;
+			Source = source;
+		}
+	}
+
+	public class EquipmentSets
+	{
+		public List<EquipmentSet> Sets;
+		private IEnumerable<Item>? ItemsSheet;
+
+		public EquipmentSets(IEnumerable<Item>? itemsSheet)
```

Comment:
And then remove nullability here (and propagate to fields &c)

Reply:
Ah, I didn't read this comment before fixing [the previous request](https://github.com/ktisis-tools/Ktisis/pull/10#discussion_r992092496)
😅 So I guess that's done too.